ibmcloud/README.md

```diff
@@ -255,82 +255,112 @@ $ install cloud-api-adaptor /usr/local/bin/
 
-The following command shows the status of the pod you just created. When it becomes running, a new peer pod VM instance is running.
-```bash
-$ kubectl get pods
+```
+ibmcloud_api_key = "<your API Key>"
+cluster_name = "<cluster name>"
 ```
 
-> **Tip:** You can check the status of pod VM instance at [https://cloud.ibm.com/vpc-ext/compute/vs](https://cloud.ibm.com/vpc-ext/compute/vs). Alternatively, you can use the `ibmcloud` command to list your images as follows.
->    ```bash
->    $ ibmcloud is instances
->    ```
+> **Notes:**
+> - `ibmcloud_api_key` is your IBM Cloud API Key that you created at [https://cloud.ibm.com/iam/apikeys](https://cloud.ibm.com/iam/apikeys).
+> - `cluster_name` is a name of a Kubernetes cluster. You must use the same value for this parameter as you used for the corresponding parameter when running the Terraform template in [ibmcloud/terraform/cluster](terraform/cluster) to create the cluster.
+
+Execute the following commands on your `development machine` to delpoy the nginx demo workload:
 
-The above YAML file also define a NodePort service. You can access the HTTP port of the pod at the worker node as follows.
 ```bash
-$ curl http://localhost:30080
+$ cd ibmcloud/terraform/run-nginx-demo
+$ terraform init
+$ terraform plan
+$ terraform apply
 ```
 
-The cloud API adaptor establishes a network tunnel between the worker and pod VMs, and the network traffic to/from the pod VM is transparently transferred via the tunnel.
+Deploying the demo workload will create a new nginx Pod and a NodePort service on your Kubernetes cluster, and a new Virtual Server instance for the peer pod will be created in your IBM Cloud VPC. The `run-nginx-demo` Terraform template will also sniff test the deployed nginx server by accessing the HTTP port of the NodePort service and test that the CPU architecture of the Kubernetes worker matches that of the peer pod instance.
+
+> **Tip:** You can run the nginx sniff test manually if you log into the Kubernetes worker node and run the command
+> ```bash
+> $ curl http://localhost:30080
+> ```
+> You can also check the CPU architecture the pod VM instance is using by running the command
+> ```bash
+> $ kubectl exec nginx -- uname -a
+> ```
+> while logged into to Kubernetes worker node. If you are using a `s390x` based image as the pod VM image, the output looks like this.
+> ```
+> Linux nginx 5.4.0-109-generic #123-Ubuntu SMP [Date] s390x GNU/Linux
+> ```
+
+> **Note:** The cloud API adaptor establishes a network tunnel between the worker and pod VM, and the network traffic to/from the pod VM is transparently transferred via the tunnel.
 
+If you execute the `terraform destroy` command for the `run-nginx-demo` Terraform template the nginx Pod, ConfigMap and NodePort Service, as well as the RuntimeClass for Kata will be deleted on the cluster.
```

Comment:
Since you raised this PR I've added a new doc section at the end which describes the clean-up process, so I think moving this there (to replace some of the current instructions) might help with the flow, as I originally did the destroy here and then the next step was the `check-podvm-instance`, which was a bit confusing.

Reply:
I moved the (Terraform) destroy instructions to separate subsections in the Clean Up section: ibmcloud#delete-the-demo-configuration-and-pod and ibmcloud#stop-the-cloud-api-adaptor-process.